changes/4617e683826ce812065e7ea82c3d1688.yaml

```diff
@@ -0,0 +1,8 @@
+---
+desc: Updated the ``Snap._applyNodeEdits()`` API to require a caller to provide the
```

Comment:
This strikes me as internal and unnecessary to include in the changelog, no?

Reply:
Removed per conversation.